cylc/flow/config.py

```diff
@@ -693,10 +695,9 @@ def process_final_cycle_point(self) -> None:
         Raises:
             WorkflowConfigError - if it fails to validate
         """
-        if (
-            self.cfg['scheduling']['final cycle point'] is not None and
-            not self.cfg['scheduling']['final cycle point'].strip()
-        ):
+        if self.cfg['scheduling']['final cycle point'] == '':
```

Comment:
The `.strip()` may or may not have been helpful, if you're totally sure it wasn't needed that's fine, otherwise perhaps:

```suggestion
        if not self.cfg['scheduling']['final cycle point'].strip():
```

Reply:
I tested out
```
final cycle point = "       "
```
and put a breakpoint at this line; the value of `self.cfg['scheduling']['final cycle point']` was just `''`. Presumably parsec  handles stripping (as I would hope so)